src/main/java/com/linagora/james/mailets/json/ClassificationRequestBody.java

```diff
@@ -82,6 +84,7 @@ private static int getSanitizedMailSize(Mail mail) throws MessagingException {
     private static Optional<String> mainTextContent(MessageContent messageContent) {
         return messageContent.getHtmlBody()
             .filter(s -> !Strings.isNullOrEmpty(s))
+            .map(jsoupHtmlTextExtractor::toPlainText)
```

Comment:
Ok, to answer you, this is not an easy question.

We have no easy way to pass the HTML  through MailboxManager API for now. It is a known point, but it requires more design, discussion, and decision. This is not easy and can't be handle as part of this PR, even if it is a known point.

(Where it get really interesting is avoid doing it for every GetMessages request)

So, it is a known point, but I won't fix it here.